docs/translations.md

```diff
@@ -0,0 +1,32 @@
+# Translation and Localization
+Translations are maintained in separate repositories that are managed by the
+[Mozilla Localization Team](https://github.com/mozilla-l10n). There is a
+Pontoon project for the
+[Relay Website](https://pontoon.mozilla.org/projects/firefox-relay-website/)
+and for the
+[Add-on](https://pontoon.mozilla.org/projects/firefox-relay-add-on/). More
+than 20 languages are supported.
+
+The translation repositories are included as submodules, such as
+`privaterelay/locales`. The submodule refers to a specific commit in the
+separate repository. A GitHub action periodically updates this to the latest
+commit, bringing in any new translations or other changes from the localization
+team.
+
+The translation bundles use the [Fluent format](https://projectfluent.org/).
+They are also included in the Docker image, and read by the email processing
+apps at runtime. They are also embedded in the JavaScript during the build
+process, so that the website text is translated.
+
+The user's desired language is stored with their Firefox Account, which uses
+the `Accept-Language` header provided by the browser. If there is a matching
```

Comment:
Hmm, this is partially true. When we forward an email to the user, we use the language that is stored in their Firefox account, which uses the Accept-Language provided by their browser _at the time they signed up for their Firefox account_, IIRC.

When they visit the website, however, we ignore that, and just use the Accept-Language that the user's browser is sending at that time, even if that differs from the language that it sent when they created their Firefox account.

Also, I'm not sure if it's worth mentioning, but there isn't always an exact match between the Accept-Language and the languages we have available, and we rely on [`@fluent/langneg`](https://github.com/projectfluent/fluent.js/tree/master/fluent-langneg) to provide us with a list of applicable locales - for example, `de-DE` if the user's Accept-Language is set to just `de`.

Reply:
I think these are the kind of details that belong here, thanks!